generator/namespaces.py

```diff
@@ -350,23 +364,33 @@ def namespaces(
 
     namespaces = {}
     for app in glean_apps:
-        looker_views = _get_looker_views(app, db_views)
-        explores = _get_explores(looker_views)
-        views_as_dict = {view.name: view.as_dict() for view in looker_views}
-
-        namespaces[app["name"]] = {
-            "owners": app["owners"],
-            "pretty_name": app["pretty_name"],
-            "views": views_as_dict,
-            "explores": explores,
-            "glean_app": True,
-        }
+        if app["name"] not in ignore:
+            looker_views = _get_looker_views(app, db_views)
+            explores = _get_explores(looker_views)
+            views_as_dict = {view.name: view.as_dict() for view in looker_views}
+
+            namespaces[app["name"]] = {
+                "owners": app["owners"],
+                "pretty_name": app["pretty_name"],
+                "views": views_as_dict,
+                "explores": explores,
+                "glean_app": True,
+            }
 
     if custom_namespaces is not None:
         custom_namespaces = yaml.safe_load(custom_namespaces.read()) or {}
+        # remove namespaces that should be ignored
+        for ignored_namespace in ignore:
+            del custom_namespaces[ignored_namespace]
```

Comment:
This needs another check, otherwise if someone passes a namespace to ignore that isn't a custom namespace this would raise a `KeyError`.
```suggestion
            if ignored_namespace in custom_namespaces:
                del custom_namespaces[ignored_namespace]
```